static/styles/compose.css

```diff
@@ -685,7 +694,7 @@ a.compose_control_button.hide {
 
     border: 1px solid hsl(0, 0%, 67%);
     border-radius: 4px;
-    background-color: hsla(0, 0%, 0%, 0.02);
+    background-color: hsl(0, 0%, 98%);
```

Comment:
Switched from transparent overlay to the exact color it was before, since the background behind it changed.